siriuspy/siriuspy/csdevice/pwrsupply.py

```diff
@@ -20,8 +20,10 @@
 # --- Wfm ---
 MAX_WFMSIZE_FBP = 1024
 DEF_WFMSIZE_FBP = 980
+DEFAULT_WFM_FBP = _np.zeros(DEF_WFMSIZE_FBP)
```

Comment:
@xresende, I think it would be nice to specify the type of the array here:
`_np.zeros(DEF_WFMSIZE_FBP, dtype=float)`